pyproject.toml

```diff
@@ -1,3 +1,187 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tmt"
+dynamic = ["version"]
+description = "Test Management Tool"
+readme = "README.rst"
+license = "MIT"
+license-files = {paths = ["LICENSE"]}
+requires-python = ">=3.9"
+authors = [
+  {name = "Petr Splichal", email = "psplicha@redhat.com"},
+]
+maintainers = [
+  {name = "Petr Splichal", email = "psplicha@redhat.com"},
+]
+keywords = [
+  "metadata",
+  "testing",
+]
```

Comment:
Can we keep consistent indent (four spaces and hanging closing brackets) in this file as well, please?

Reply:
Thanks, I just moved the brackets in 360d2b0e.